scss/composer/_composer.scss

```
@@ -25,7 +23,11 @@
     & > .attachment-quote-section {
       display: flex;
       justify-content: space-between;
-      padding: 3px 0;
+      padding: 5px 0 0 5px;
+      background-color: #efefef;
+      border-radius: 8px;
+      margin: 0px 12px 10px;
+      opacity: 0.85;
```

as a minimal thing to solve the issues at hand, the following would do:

```suggestion
      padding: 9px;
      border-bottom: 1px solid #eee;      
```

for quotes, i would not do "more".

by more "room" around the quotes, things look nicer already.

for vcards, a rounded box and a slighly greyish background would be nice, however.

but i would leave the "X" button outside the box and also let the box appear not at 100% wide (this does not look like a "card" at the end)

but as said, also without, it fixes the issue at hand and is an improvement.

---

@r10s  Applied your suggestions. Feedback welcome

---

thanks a lot! ❤️